<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验回溯系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_import">
       <attribute name="title">
        <string>数据导入</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="csv_group">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>数据文件读取</string>
          </property>
          <layout class="QVBoxLayout" name="csv_layout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="csv_select_layout">
             <item>
              <widget class="QPushButton" name="btn_add_csv">
               <property name="text">
                <string>选择数据文件夹</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="csv_path_line">
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="csv_log">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>请确保所有的数据文件放在一个文件夹下</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="videoGroup">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>视频文件读取并保存于本地</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="nvr_lineEdit">
               <property name="placeholderText">
                <string>nvr读取协议</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nvr_testBtn">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="camera_lineEdit">
               <property name="placeholderText">
                <string>高速摄像机协议</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="camera_testBtn">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="video_save_path_lineEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>视频文件保存路径</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="video_save_path_browseBtn">
               <property name="text">
                <string>浏览</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QProgressBar" name="video_collect_progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="video_collectBtn">
               <property name="text">
                <string>开始导出</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="video_save_log"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="output_group">
          <property name="title">
           <string>输出设置</string>
          </property>
          <layout class="QVBoxLayout" name="output_layout">
           <item>
            <layout class="QHBoxLayout" name="output_select_layout">
             <item>
              <widget class="QLineEdit" name="db_save_path_lineEdit">
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>选择数据库保存地址</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="db_save_pathBtn">
               <property name="text">
                <string>浏览...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dbCreate">
               <property name="text">
                <string>生成数据库文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_import">
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text_import_log">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>导入日志将在此显示...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_display">
       <attribute name="title">
        <string>实验回放</string>
       </attribute>
       <layout class="QVBoxLayout" name="display_layout">
        <item>
         <layout class="QHBoxLayout" name="main_content_layout">
          <item>
           <layout class="QVBoxLayout" name="video_layout">
            <property name="spacing">
             <number>5</number>
            </property>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="videoArea">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>单视角</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>主视角</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QFrame" name="frame_3">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QFrame" name="frame_4">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QFrame" name="frame_5">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>全局视角</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QFrame" name="frame_6">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QFrame" name="frame_7">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QFrame" name="frame_8">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QFrame" name="frame_9">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QFrame" name="frame_10">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QFrame" name="frame_13">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QFrame" name="frame_12">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QFrame" name="frame_11">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QFrame" name="frame_14">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QFrame" name="frame_17">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QFrame" name="frame_16">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QFrame" name="frame_15">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QFrame" name="frame_18">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QFrame" name="frame_21">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QFrame" name="frame_20">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="QFrame" name="frame_19">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="data_panel_layout">
            <item>
             <widget class="QTabWidget" name="tab_data_panel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="tab_status_data">
               <attribute name="title">
                <string>状态数据</string>
               </attribute>
               <layout class="QVBoxLayout" name="status_layout">
                <item>
                 <widget class="QTableWidget" name="table_status_data">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_experiment_data">
               <attribute name="title">
                <string>实验数据</string>
               </attribute>
               <layout class="QVBoxLayout" name="experiment_layout">
                <item>
                 <widget class="QTableWidget" name="table_experiment_data">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_annotations">
               <attribute name="title">
                <string>标注列表</string>
               </attribute>
               <layout class="QVBoxLayout" name="annotations_layout">
                <item>
                 <widget class="QTableWidget" name="table_annotations">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="annotation_buttons">
                  <item>
                   <widget class="QPushButton" name="btn_edit_annotation">
                    <property name="text">
                     <string>编辑</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btn_delete_annotation">
                    <property name="text">
                     <string>删除</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="annotation_spacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="controls_layout">
          <item>
           <layout class="QHBoxLayout" name="playback_controls">
            <item>
             <widget class="QPushButton" name="btn_play_pause">
              <property name="text">
               <string>播放/暂停</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slider_position">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_time">
              <property name="text">
               <string>00:00 / 00:00</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="playSpeedControl"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_export">
       <attribute name="title">
        <string>实验导出</string>
       </attribute>
       <layout class="QVBoxLayout" name="export_layout">
        <item>
         <widget class="QLabel" name="label_export_placeholder">
          <property name="text">
           <string>导出功能开发中...</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
